src/components/Cv/index.js

```diff
@@ -1,5 +1,68 @@
 import { h } from 'preact';
+import { Link } from 'preact-router/match';
 
-const Cv = () => <div>My CV hola hola</div>;
+import CloseButton from 'assets/icons/CloseButton.js';
+
+import s from './styles.scss';
+
+const Cv = () => (
+  <div className={s.cv}>
+    <Link href="/" className={s.closeButton}>
+      <CloseButton />
+    </Link>
+    <div className={s.idWrapper}>
+      <h2 className={s.idTitile}>Bruna Brakus</h2>
+      <p className={s.idDescription}>
+        Split, Croatia
+        <br />brunabrakus@protonmail.com
+      </p>
+    </div>
+
+    <div className={s.about}>
+      <div className={s.card}>
```

Comment:
`Card` should be component used with passed data, similiar to `Work` component with  `examples.js`